pkg/services/pluginsintegration/clientmiddleware/logger_middleware.go

```diff
@@ -235,3 +234,10 @@ func (m *LoggerMiddleware) ConvertObjects(ctx context.Context, req *backend.Conv
 
 	return resp, err
 }
+
+func statusSourceFromPluginErrorSource(pluginErrorSource backend.ErrorSource) backend.ErrorSource {
+	if !pluginErrorSource.IsValid() {
+		return backend.DefaultErrorSource
+	}
+	return pluginErrorSource
+}
```

Comment:
quite small but we could move this to the sdk so the logic is not duplicated

Reply:
Fixed